src/relayFeeCalculator/chain-queries/baseQuery.ts

```diff
@@ -1,7 +1,7 @@
 import { SpokePool, SpokePool__factory } from "@across-protocol/contracts-v2";
 import { L2Provider } from "@eth-optimism/sdk";
 import { providers } from "ethers";
-import Coingecko from "../../coingecko/Coingecko";
+import { Coingecko } from "../../coingecko";
```

Comment:
@james-a-morris This fixes an issue introduced in https://github.com/across-protocol/sdk-v2/pull/80 that's fialing lint on master